torch/csrc/Storage.cpp

```diff
@@ -353,6 +404,14 @@ PyTypeObject THPStorageType = {
     THPStorage_pynew, /* tp_new */
 };
 
+int THPStorageMetaType_init(PyObject* cls, PyObject* args, PyObject* kwargs) {
+  if (PyType_Type.tp_init(cls, args, kwargs) < 0) {
+    return -1;
+  }
+  ((PyTypeObject*)cls)->tp_dealloc = (destructor)THPStorage_subclass_dealloc;
```

Comment:
It's worth noting that this is substantially less important for Storage, since we don't really have storage subclasses lol.

Reply:
I think it's still required since `torch.UntypedStorage` subclasses `torch._C.StorageBase`

Reply:
Yup